### PR TITLE
Update to TypeScript 3.7.2

### PR DESCRIPTION
**Pull request by snario**
This also updates some `redux-saga` packages because they were not compatible.

### CODE REVIEW

**Comment by snario** [comment]
(work in progress)

File: packages/rps/src/redux/message-service/firebase-inbox-listener.ts [hunk 1]
```diff
@@ -10,7 +10,8 @@ export function* firebaseInboxListener(client: RPSChannelClient) {
     reduxSagaFirebase.database.channel,
     `/messages/${address.toLowerCase()}`,
     'child_added',
-    buffers.fixed(10)
+    // TODO: @george why was buffers added here?
+    // buffers.fixed(10)
```
Comment (snario) [inline packages/rps/src/redux/message-service/firebase-inbox-listener.ts]: `call` doesn't have an argument for this at the end, so I am a bit confused. It was throwing an error after upgrading TypeScript. @geoknee

Comment (geoknee) [reply packages/rps/src/redux/message-service/firebase-inbox-listener.ts]: For context: You also updated `redux-saga`.

I'm not sure why this is here, though. Looks like it was probably my mistake: perhaps I saw `channel` on L10 and read `actionChannel` https://redux-saga.js.org/docs/api/#actionchannelpattern-buffer which does take a buffer argument.

Comment (geoknee) [reply packages/rps/src/redux/message-service/firebase-inbox-listener.ts]: I think you're good to just remove it.

**Comment by snario** [comment]
Blocked on #602

**Pull request merged**